org.uqbar.project.wollok.launch/src/org/uqbar/project/wollok/launch/tests/StackTraceElementDTO.xtend

```diff
@@ -0,0 +1,45 @@
+package org.uqbar.project.wollok.launch.tests
+
+import java.io.File
+import java.io.Serializable
+import org.eclipse.emf.common.util.URI
+import org.eclipse.xtend.lib.annotations.Accessors
+
+@Accessors
+class StackTraceElementDTO implements Serializable {
+	String contextDescription
+	Integer lineNumber
+	String fileName
+	
+	new(String contextDescription, String fileName, int lineNumber) {
+		this.contextDescription = contextDescription
+		this.fileName = fileName
+		this.lineNumber = lineNumber
+	}
+	
+	def toLink(URI testResource) {
+		val result = new StringBuffer
+		result.append("   ")
+		result.append("at ") 
+		if (contextDescription != null) {
+			result.append(contextDescription)
+			result.append(" ")
+		}
+		if (fileName != null && !fileName.isEmpty) {
+			result.append("[<a href=\"" + fileName + ":" + lineNumber + "\">" + fileName.location(lineNumber) + "</a>]\n")
```

Comment:
ho.. cool.

Didn't know that the links on the console where intepreted from html code.